### PR TITLE
Pointwatch -- gold/golden mismatch

### PR DESCRIPTION
**Pull request by trv6**
Changed instances of 'golden' to 'gold' to match the ReadMe.

### CODE REVIEW

**Comment by trv6** [comment]
Gold -> Golden instead

File: addons/pointwatch/statics.lua [hunk 1]
```diff
@@ -142,7 +142,7 @@ function initialize()
         setfenv(cur_func,_G)
         dynamis.entry_time = os.clock()
         dynamis.zone = info.zone
-        windower.add_to_chat(123,'Loading PointWatch in Dynamis results in an inaccurate timer. Number of KIs is displayed.')
+        error(123,'Loading PointWatch in Dynamis results in an inaccurate timer. Number of KIs is displayed.')
```
Comment (z16) [inline addons/pointwatch/statics.lua]: The `123` needs to be removed for this to work right. Was this change intended? This doesn't sound like an error.

**Comment by trv6** [comment]
@z16  Unintended. I may have edited the live file and replaced the dev file with it.
